healthd/include_charger/charger/healthd_mode_charger.h

```diff
@@ -116,6 +116,7 @@ class Charger {
     int64_t next_key_check_ = 0;
     int64_t next_pwr_check_ = 0;
     int64_t wait_batt_level_timestamp_ = 0;
+    int32_t min_battery_for_boot_{8}; // percentage
```

Comment:
int32_t min_battery_for_boot_{MIN_BATTERY_FOR_BOOT};

Reply:
Prefer to leave it since there is a potential unintended misuse of the macro somwhere.